src/main/java/racingGame/domain/Car.java

```diff
@@ -0,0 +1,28 @@
+package racingGame.domain;
+
+public class Car {
+    private final int MOVE_CONDITION = 4;
```

Comment:
상수는 `static final` 필드입니다. 아래의 코드로 개선해 보면 어떨까요?

```suggestion
    private static final int MOVE_CONDITION = 4;
```